modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -34,4 +34,43 @@ Before you deploy {lvms} on {sno} clusters, be aware of the following limitation
 
 * You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
 * You can make only a single `deviceClass` entry in the `LVMCluster` CR.
-* When a device becomes part of the `LVMCluster` CR, it cannot be removed.
\ No newline at end of file
+* When a device becomes part of the `LVMCluster` CR, it cannot be removed.
+
+[id="lvms-deployment-limitations-for-sno-ran_{context}"]
+== Limitations
+
+For deploying {sno}, LVM Storage has the following limitations:
+
+* The total storage size is limited by the size of the underlying Logical Volume Manager (LVM) thin pool and the overprovisioning factor.
+* The size of the logical volume depends on the size of the Physical Extend (PE) and the Logical Extend (LE).
+** It is possible to define the size of PE and LE during the physical and logical device creation.
+** The default PE and LE size is 4 MB.
+** If the size of the PE is increased, there are almost no limits for the size of LVM.
```

Comment:
> there are almost no limits for the size of LVM

We have to be more clear on this because "almost no limits" implies that there is still some kind of limit. Could you share more information about this?